unifi/port_conf.go

```diff
@@ -0,0 +1,25 @@
+package unifi
+
+import (
+	"context"
+)
+
+func (c *Client) ListPortConf(ctx context.Context, site string) ([]PortConf, error) {
```

Comment:
Do you think we should rename these to `SwitchPort` or something, the `PortConf` name is not entirely straightfoward, confused with `PortForward` easily I think.

Reply:
How about `PortProfile`? I think it makes the most sense (since this can also setup ports on routers, APs, etc, not just switches)

Reply:
Sounds good 👍